src/model/modifier/DraftModifier.js

```diff
@@ -26,6 +26,8 @@ var invariant = require('invariant');
 var removeEntitiesAtEdges = require('removeEntitiesAtEdges');
 var removeRangeFromContentState = require('removeRangeFromContentState');
 var setBlockTypeForContentState = require('setBlockTypeForContentState');
+var setBlockDataForContentState = require('setBlockDataForContentState');
+var mergeBlockDataForContentState = require('mergeBlockDataForContentState');
 var splitBlockInContentState = require('splitBlockInContentState');
 
 import type {BlockMap} from 'BlockMap';
```

Comment:
Ok I will change that. I'am off this week.

On Tue, 22 Mar 2016 16:41 Isaac Salier-Hellendag, <notifications@github.com>
wrote:

> In src/model/modifier/DraftModifier.js
> <https://github.com/facebook/draft-js/pull/216#discussion_r57009690>:
>
> > @@ -26,6 +26,8 @@ var invariant = require('invariant');
> >  var removeEntitiesAtEdges = require('removeEntitiesAtEdges');
> >  var removeRangeFromContentState = require('removeRangeFromContentState');
> >  var setBlockTypeForContentState = require('setBlockTypeForContentState');
> > +var setBlockDataForContentState = require('setBlockDataForContentState');
> > +var mergeBlockDataForContentState = require('mergeBlockDataForContentState');
> >  var splitBlockInContentState = require('splitBlockInContentState');
> >
> >  import type {BlockMap} from 'BlockMap';
>
> Instead of requiring Map above, it can be imported as a Flow type.
>
> import type {Map} from 'immutable';
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/draft-js/pull/216/files/4d3da5882d909b5613248a9815500fbdae318f4b..45cec09537ed0bb2e09288223937b93bd061b4a5#r57009690>
>